prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -153,10 +155,18 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
           this.enrolmentFormStateService.setForm(enrolment);
 
           this.enrolmentErrors = this.getEnrolmentErrors(enrolment);
+          // Determine if Enrolment is within 90 days of expiration
+          this.enrolmentWithin90DaysOfExpiry();
         })
       ).subscribe();
   }
 
+  public enrolmentWithin90DaysOfExpiry(): void {
+    const enrolment = this.enrolmentService.enrolment;
+    const expiryDate = enrolment.expiryDate;
+    this.withinDaysOfRenewal = DateUtils.withinDaysBeforeDate(expiryDate, 90) ? true : false;
```

Comment:
`withinDaysBeforeDate` already returns `boolean` so conditional operator is not needed.